src/tools/jsondocck/src/main.rs

```diff
@@ -101,10 +101,10 @@ fn get_commands(template: &str) -> Result<Vec<Command>, ()> {
             continue;
         };
 
-        if let Some((kind, path)) = CommandKind::parse(&cap["cmd"], negated, &args) {
-            commands.push(Command { kind, lineno, path: path.to_owned() })
+        if let Some((kind, path)) = DirectiveKind::parse(&cap["cmd"], negated, &args) {
```

Comment:
Should the named capture (`cmd`) group not also be renamed?

Reply:
Yes, Good catch, Thanks. https://github.com/rust-lang/rust/compare/508479d9c118924b3891132b35ce6106a3020a25..14db1b5b1c2b55dfa89c7b958e70efb936a33e7c